amp-story/templates/ampmars/index.html

```diff
@@ -0,0 +1,967 @@
+        "@type": "CreativeWork",
+        "headline": "The Mars Cruise",
+        "url": "https://www.example.com"
+      }
+    </script>
+    <style amp-custom>
+      :root {
+        --yellow: #ffd200;
+        --bl-0: rgba(0, 0, 0, 0);
+        --bl-1: rgba(0, 0, 0, 1);
+      }
+
+      amp-story {
+        color: #eaeaea;
+        font-weight: 300;
+        font-family: 'Roboto', sans-serif;
+      }
+
+      amp-story .horiz {
+        display: flex;
+        flex-direction: row;
+      }
+      amp-story .horiz::before,
+      amp-story .horiz::after {
+        content: "";
+        flex: 1 1;
+        border-bottom: 0.0625em solid #fff;
+        margin: auto;
+      }
+      amp-story .horiz::before {
+        margin-right: 2.3rem;
+      }
+      amp-story .horiz::after {
+        margin-left: 2.3rem;
+      }
+
+      amp-story h2 {
+        color: var(--yellow);
+        font-size: 6.8vw;
+        font-weight: 700;
+        font-family: 'Josefin Sans', sans-serif;
+        text-transform: uppercase;
+        margin-bottom: -2vw;
+      }
+
+      amp-story h3 {
+        font-weight: 700;
+        font-family: 'Josefin Sans', sans-serif;
+      }
+
+      amp-story p {
+        font-size: 3.3vw;
+        line-height: 4.9vw;
+        letter-spacing: 0.1vw;
+      }
+
+      amp-story .yellow {
+        color: var(--yellow);
+      }
+
+      /* PAGE 01: COVER - THE MARS CRUISE */
+      #cover {
+        background-color: #71311c;
+      }
+
+      #cover[active] .bkgd {
+        animation-name: coverPan;
+        animation-duration: 30s;
+        animation-iteration-count: 1;
+        animation-direction: normal;
+        animation-fill-mode: forwards;
+        animation-play-state: running;
+        animation-timing-function: ease-out;
+      }
+
+      #cover .gradient {
+        background: linear-gradient(to bottom, var(--bl-0) 0%, var(--bl-0) 34%, var(--bl-1) 100%);
+      }
+      #cover .copy-layer {
+        margin: 0 auto;
+        padding: 0 14vw 12vw;
+        align-content: flex-end;
+      }
+      #cover h1 {
+        color: #fff;
+        font-size: 19.4vw;
+        font-weight: 700;
+        font-family: 'Josefin Sans', sans-serif;
+        line-height: 20vw;
+      }
+      #cover .horiz {
+        padding: 0 2vw;
+        font-size: 7.5vw;
+        font-weight: 700;
+        font-style: italic;
+        font-family: 'Tienne', serif;
+      }
+      #cover .yellow {
+        font-size: 25vw;
+        display: block;
+      }
+
+      #cover p {
+        font-size: 4.2vw;
+        line-height: 6.3vw;
+        text-align: left;
+        padding: 0 3vw;
+      }
+
+      /* PAGE 02: THE JOURNEY */
+      #journey {
+        background-color: black;
+      }
+
+      #journey .copy-layer {
+        padding: 12.8vw 21vw;
+      }
+
+      #journey h2 {
+        margin-bottom: -1vw;
+      }
+
+      /*
+      * PAGE 04: TRIP HIGHLIGHTS
+      * Stylelint requires `#trip-highlights` styles before `#what-to-bring`
```

Comment:
We can relax that restriction.  I'm out of the office this week, but can hopefully address that next week.